### ISSUE TITLE
Dashboard does not show correctly history status of binary sensors

### ISSUE THREAD

**Issue by andrejpan** [home-assistant/frontend#23084]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

We have around 20 Agara [window sensors](https://www.aqara.com/eu/product/door-and-window-sensor/) connected to HA with conbee3 gateway in ZHA integration. 

We have a few dashboards which present the status of these sensors.
![Screenshot from 2024-12-01 19-16-49](https://github.com/user-attachments/assets/99361277-1fe1-40e2-bc7c-1648a2201407)

We show the state of these devices, the last update, and the last change in the Apperance menu. 
![Screenshot from 2024-12-01 19-00-55](https://github.com/user-attachments/assets/1d03ae05-a039-49ee-a5b8-bedcc38c316e)

Now we observe two issues. One case is that all devices were updated (and changed 6 hours ago) and two devices here are unavailable. Please see the first image and the image below.
![Screenshot from 2024-12-01 19-22-30](https://github.com/user-attachments/assets/dc7794f9-ef6f-4670-997e-6305d22868b8)

According to the logs, sensor `03_dining_room_door_sensor_right_opening` has been unavailable for three days. Why does the Dashboard say that there was a change six hours ago?

A similar case is with available sensors. The logs of `02_living_room_door_sensor_left_opening` state that there was a change yesterday evening (24 hours ago), but the dashboard states that the change was 6 hours ago.  
![Screenshot from 2024-12-01 19-26-24](https://github.com/user-attachments/assets/3cb7fefa-06b1-42ff-b7d0-8ec7c5633a1a)

### Describe the behavior you expected

The dashboard should show the timestamps of binary_sensor (last update and last change) according to data in the Logbook.

### Steps to reproduce the issue

Tile Card configuration for one sensor

```
type: tile
entity: binary_sensor.02_living_room_door_sensor_1_opening
grid_options:
  columns: 12
  rows: 1
tap_action:
  action: none
hold_action:
  action: more-info
state_content:
  - state
  - last_updated
  - last_changed
```

### What version of Home Assistant Core has the issue?

2024.11.3

### In which browser are you experiencing the issue with?

Brave - Version 1.73.91 Chromium: 131.0.6778.85 (Official Build) (64-bit)

### Which operating system are you using to run this browser?

Ubuntu 24.04

**Comment by ildar170975** [comment]
When was the last time you rebooted HA?

**Comment by andrejpan** [comment]
A bit more than 1 day ago.
![Screenshot from 2024-12-01 21-45-08](https://github.com/user-attachments/assets/1df5efb4-c685-4312-a5e7-a98f349d03e9)

I do not fully understand your last sentence. When HA is restarted, "last-changed & last-updated" of all sensors should also be updated?

**Comment by ildar170975** [comment]
> When HA is restarted, "last-changed & last-updated" of all sensors should also be updated?

This binary sensor is always ON. HA was restarted ~2 hours ago.
![image](https://github.com/user-attachments/assets/04f512d4-51ce-4d0d-9303-ad6ed632cf98)

**Comment by andrejpan** [comment]
![Screenshot from 2024-12-01 22-07-04](https://github.com/user-attachments/assets/fa6f83a8-c0a1-4f85-928d-7c0ae9915c48)
I also restarted my HA, logbook does not have any updates for these sensors.

**Comment by ildar170975** [comment]
> logbook does not have any updates for these sensors.

But last-changed is updated to "~ time of HA started", right?

**Comment by andrejpan** [comment]
Yeah, both timestamps in the Dashboard are equal to "~ time of HA started;" however, that does not make much sense to me. The information about both changes in the Logbook are still old.

**Comment by ildar170975** [comment]
> The information about both changes in the Logbook are still old.

1. Logbook contains valid times for every change of a state.
2. A value of `last-changed` is persistent if HA is not rebooted. Otherwise this value is updated on HA startup. If you expect a different behaviour - register a "Feature request" (or join an already present one in a "Feature requests" section of Community).
